docs/deploying_ccm_and_csi_with_multi_dc_vc_aka_zones.md

```diff
@@ -0,0 +1,187 @@
+
+In the example `vsphere.conf` below, `k8s-region` and `k8s-zone` was selected:
+
+```
+[Global]
+# properties in this section will be used for all specified vCenters unless overridden in VirtualCenter section.
+
+user = "vCenter username for cloud provider"
+password = "password"
+port = "443" #Optional
+insecure-flag = "1" #set to 1 if the vCenter uses a self-signed cert
+datacenters = "list of datacenters where Kubernetes node VMs are present"
+
+[VirtualCenter "1.2.3.4"]
+# Override specific properties for this Virtual Center.
+        user = "vCenter username for cloud provider"
+        password = "password"
+        # port, insecure-flag, datacenters will be used from Global section.
+
+[VirtualCenter "10.0.0.1"]
+# Override specific properties for this Virtual Center.
+        port = "448"
+        insecure-flag = "0"
+        # user, password, datacenters will be used from Global section.
+
+[Labels]
+region = k8s-region
+zone = k8s-zone
+```
+
+Configure your vsphere.conf file and create a `configmap` of your settings using the following command:
+
+```bash
+[k8suser@k8master ~]$ kubectl create configmap cloud-config --from-file=vsphere.conf --namespace=kube-system
+```
+
+#### 1. Creating Zones in your vSphere Environment via Tags
+
+ The `region` tag is just a construct that allows one to make a grouping for a specific set of resources. It could be used to indicate something like a geographic location like a country or perhaps a specific datacenter. This label is an arbitrary grouping that you decide on. The `zone` tag is another construct that allows you to further subdivide resources within a `region`. As an example, using the countries as a `region`, the `zone` could indicate a specific datacenter out of a list in that `region`. In the second example of using a datacenter as a `region`, you might use a `zone` to indicate a specific rack within the datacenter or even just a cluster within that datacenter. Then all hosts and subsequently all VMs acting as Kubernetes worker nodes under that tagged datacenter or cluster inherit the tags of those parent objects. How one chooses to group regions and zones is completely based on how you want to identify a specific group of resources.
+
+There are many options for creating vSphere tags. One such method would be to use [govc](https://github.com/vmware/govmomi/tree/master/govc). All the examples below will make use of this method. You could also create tags by accessing the vSphere REST APIs directly or by using the vSphere UI.
+
+Using the example above, if it is decided that `k8s-region` and `k8s-zone` are to be used for your Category labels, then you can create those vSphere Categories using `govc` by running the following command:
+```bash
+[k8suser@k8master ~]$ GOVC_URL=https://REPLACE_VSPHERE_USERNAME:REPLACE_VSPHERE_PASSWORD@REPLACE_VSPHERE_IP/sdk GOVC_INSECURE=TRUE ./govc tags.category.create -d "Kubernetes region" k8s-region
```

Comment:
Instead of using inline env vars, why not exporting them at the beginning of the example? maybe removing `GOVC_INSECURE=TRUE` which is not best practice.

Reply:
Will do.